content/kubermatic/master/tutorials/13-psp/_index.en.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Pod Security Policy"
+date = 2020-04-02T12:07:15+02:00
+weight = 130
++++
+
+[Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) is a key security feature in Kubernetes. It allows cluster administrators to set [granular controls](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#policy-reference) over security sensitive aspects of pod and container spec.
+
+PSP is implemented using an optional admission controller that's disabled by default. It's important to have an initial authorizing policy on the cluster _before_ enabling the PSP admission controller, especially on existing existing cluster is important. Without an authorizing policy, the the controller will prevent all pods from being created on the cluster.
```

Comment:
> especially on existing existing cluster is important

The grammar seems to be somewhat out here. Also "cluster" should probably be plural?

> the the controller 

One "the" too many.